symphony/content/content.publish.php

```diff
@@ -37,33 +37,31 @@ public function sort(&$sort, &$order, $params)
             EntryManager::setFetchSorting($sort, $order);
         } else {
             $sort = General::sanitize($sort);
+            $filters = '';
 
             // Ensure that this field is infact sortable, otherwise
             // fallback to IDs
             if (($field = FieldManager::fetch($sort)) instanceof Field && !$field->isSortable()) {
                 $sort = $section->getDefaultSortingField();
             }
 
+            // Format the filter query string
+            if ($params['filters']) {
+                $filters = preg_replace('/^&amp;/i', '', $params['filters'], 1);
```

Comment:
> Wouldn't it be simpler

it's trim vs preg_replace....

> faster

not for a one time call

> bulletproof

Totally not, since, like stated in the php's doc, it trims all '&' OR 'a' OR 'm' OR 'p' OR ';', not the characters in the given order...

So a value of '&amp;aa' would still comes out empty.